### ISSUE TITLE
Activation Support Tracker

### ISSUE THREAD

**Issue by narendasan** [NVIDIA/TRTorch#10]
Need to support:
- [x] ReLU
- [x] tanh
- [x] hardtanh
- [x] sigmoid
- [ ] Leaky ReLU
- [ ] eLU
- [ ] SeLU
- [ ] SoftSign
- [ ] SoftPlus
- [ ] ReLU6
- [ ] GeLU
- [ ] PReLU

Relevant Documentation:
- https://docs.nvidia.com/deeplearning/sdk/tensorrt-archived/tensorrt-601/tensorrt-api/c_api/classnvinfer1_1_1_i_network_definition.html#a0382282a59e3841726f6c29c4ac1f684

- https://docs.nvidia.com/deeplearning/sdk/tensorrt-archived/tensorrt-601/tensorrt-api/c_api/classnvinfer1_1_1_i_activation_layer.html

- https://pytorch.org/docs/stable/nn.functional.html#non-linear-activation-functions

- https://github.com/pytorch/pytorch/blob/12fb8148e48d3e39d16832a84ac4b59182a00c4e/aten/src/ATen/native/native_functions.yaml#L2337

**Comment by narendasan** [comment]
Related #150
